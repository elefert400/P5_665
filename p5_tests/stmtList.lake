void foo(){}
int main(){
    int x;
    bool y;
}