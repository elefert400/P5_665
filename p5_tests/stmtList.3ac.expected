[BEGIN GLOBALS]
[END GLOBALS]
[BEGIN foo LOCALS]
[END foo LOCALS]
enter foo
lbl_0: leave foo
[BEGIN main LOCALS]
x (local)
y (local)
[END main LOCALS]
enter main
lbl_1: EXIT